### ISSUE TITLE
Web browser

### ISSUE THREAD

**Issue by owlandfox** [oculus-samples/Unreal-SharedSpaces#33]
This isnt directly related to the Shared Spaces code as such and is more of a general Oculus X Unreal issue but I am trying to use the native web browser in my project for Ready Player Me avatar customisation. Works fine in editor but just displays a blank screen on device. After scouring a few forums a few people summised that Oculus does not allow accessing urls like this for security reasons etc. But I see plenty other apps accessing urls internally just fine - the DeoVR media player for one. Can you confirm the official stance on this? Is there a way we can setup the web browser by maybe setting some whitelisted urls to appease security qualms?

**Comment by els-oculus** [comment]
Do you mean something like Launch URL?

![launch](https://user-images.githubusercontent.com/61444568/183780516-e3501cd1-0b43-48c6-bf61-a542a373d5cb.png)

That will send you back to Home to the browser.  It works for me.   Do you mean some specific URLs are blocked?

**Comment by owlandfox** [comment]
I was wanting to keep the user inside our app and present them with web page using unreal web browser in UMG but all im getting is a blank screen when applying the web page inside UMG on device.

**Comment by pr0-zac** [comment]
I took a look at the Web Browser Widget plugin. I'm not able to get it working on device either. In RenderDoc I can see the widget doesn't attempt to render at all. 

I'm not familiar enough with the internal workings of the Slate UI system to see where it's failing, but I expect the relevant class is in Engine\Source\Runtime\WebBrowser\Private\Android\AndroidWebBrowserWidget.cpp. Would need to figure out why the texture is not created and passed along to be rendered. 

I'll post the same info in an existing post on the developer forums as this issue isn't relevant to the SharedSpaces project. https://forums.oculusvr.com/t5/Unreal-VR-Development/Problems-with-using-WebBrowser-plugin-for-Unreal-Engine-4/td-p/864802

**Comment by els-oculus** [comment]
I just did a quick test here in both UE4 and UE5.   Works on Rift/Windows.

![ue5_windows](https://user-images.githubusercontent.com/61444568/184038916-7b5423ed-bb35-4b00-9d1a-9e407e0b6edc.png)


In the case of UE4 I also did both Vulkan and OpenGL on Quest, and the screen is either white or black.
In UE5, it's just Vulkan, and it's white.

Looks like that Web Browser component from Epic is broken.  Oculus doesn't block URLs.   Test it with the web browser in Home.

I'm going to close this, as this is not SharedSpaces-specific, but that is an interesting issue.   We will keep an eye on the forum as pointed out by pr0-zac.
